0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -66,10 +72,11 @@ The SPSP Endpoint MUST respond to HTTPS `GET` requests in the following manner:
 The sender queries the SPSP endpoint to get information about the type of payment that can be made to this receiver:
 
 #### Request
+
 ``` http
 GET /api/spsp/bob HTTP/1.1
 Host: red.ilpdemo.org
-Accept: application/json
+Accept: application/x-spsp-response
```

Comment:
Should be `Accept: application/spsp+json` and we should get a [provisional registration](https://tools.ietf.org/html/rfc6838#section-5.2.1).